### DISCUSSION TITLE
What do you think about pnpm?

### DISCUSSION THREAD

**Discussion by andrecasal** [epicweb-dev/epic-stack#262] [Ideas]
I've been giving pnpm a try lately. Install times are much faster due to its centralized store. It also seems possible to cache (and prune) npm packages with GitHub Actions.

What do you think about pnpm?

**Comment by andrecasal** [comment]
This might be useful: https://github.com/pnpm/action-setup

**Comment by kentcdodds** [comment]
I've never used pnpm primarily because I've not had issues with npm being slow enough to justify wanting to use another non-standard tool. So I don't plan on switching to pnpm.
